### ISSUE TITLE
Sample DS File

### ISSUE THREAD

**Issue by Seiha106** [openvpi/DiffSinger#21]
I think it would be good to add the sample DS file to the repo

**Comment by yqzhishen** [comment]
Sample DS files are added in the `samples` directory. We currently use [opensvip](https://github.com/yqzhishen/opensvip) to obtain DS files, but that tool has only Chinese version. Note that DS file structures may change, and old files may break in the future.
A public release containing experimental pretrained models are also provided for usage of different DS files. To use DS files and these models, please read code in `main.py` carefully.
We normally share our updates in communities in China (for example, Bilibili and QQ groups). Sorry for the lack of documentation on GitHub! Feel free to leave more comments if you have questions with DS files, code usage and experimental models.

**Comment by Seiha106** [comment]
I am very interested in diffsinger, but I think it is very disappointing to share information in a limited space such as QQ
Don't you want to create an information sharing community through GitHub and Discord other than QQ?

**Comment by yqzhishen** [comment]
Yes, we did realize that we did not share enough information on a global public community. There are some major reasons for this:
- Most of our people have difficulties to visit Discord (or even GitHub) here in China.
- We are doing a lot of experiments on this codebase, from data labeling/preprocessing to hyper parameter tuning. Most of the experiments haven't reached a desirable result, and we got no idea how to share them.
- We don't have much time to share our detailed daily progress. We have several people here to help with the experiments (for example, trying out hyper parameters). QQ is a limited space, but it is more efficient and real-time for our group.

But there is exactly something to share:
- A new vocoder with 44.1k sample rate has been trained and will be released in the next few days. Updates and releases of vocoders will be shared on [this page](https://openvpi.github.io/vocoders).
- A brand new approach for potentially training SVS models without labeling is being experimented and tested. See the [diff-svc](https://github.com/prophesier/diff-svc) repo for more details (we are in deep cooperation with owner and maintainers of that repo, too).
- A revised version of opencpop data labels and the corresponding preprocessing code can be released in the future.

For international followers of DiffSinger like you, and anyone who would like to provide their help (for example, ideas of improving the network, etc., or GPU resources to help trying out hyper parameters), it is highly recommended to create a topic on GitHub Discussions, as we ourselves will create some in the future.
